justfile

```diff
@@ -13,9 +13,18 @@ init:
   pushd typescript-go && git am --3way --no-gpg-sign ../patches/*.patch && popd
   mkdir -p internal/collections && find ./typescript-go/internal/collections -type f ! -name '*_test.go' -exec cp {} internal/collections/ \;
 
+init-win:
```

Comment:
copy oxc?

https://github.com/oxc-project/oxc/blob/f897e303eb3ad54a78f5d38e3543c2fc2d96951b/justfile#L90-L96

and then devs just have to run `init` not `init-win` or `init` for unix